result/bmpprefix.yaml

```diff
@@ -0,0 +1,652 @@
+components:
+  schemas:
+    BmpPrefix.State.Request:
+      description: >-
+        The request to retrieve BMP server learned BGP peer prefix information.
+      type: object
+      properties:
+        peer_bgp_id:
```

Comment:
What if two BMP servers are connected to same BGP peer ... this probably needs to have BMP server name and then array of peers with route info.